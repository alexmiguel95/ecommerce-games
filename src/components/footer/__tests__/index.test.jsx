import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../index';

it("Footer da pagina", () => {
	const tree = renderer.create(
        <Footer>
            <div>
                <div>
                    <figure>
                        <img />
                    </figure>
                    <span>&copy; text <a>text</a></span>
                </div>
            </div>
        </Footer>
    ).toJSON();

	expect(tree).toMatchSnapshot();
});
