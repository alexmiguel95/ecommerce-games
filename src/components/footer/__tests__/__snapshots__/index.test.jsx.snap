// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer da pagina 1`] = `
<footer
  className="sc-dlfnbm YoOzv"
>
  <div
    className="sc-bdfBwQ jFredM"
  >
    <header
      className="sc-gsTCUz vgldJ"
    >
      <figure>
        <img
          alt="Logo Challenge"
        />
      </figure>
      <span>
        Â© 2021 - Created by 
        <a
          href="https://www.linkedin.com/in/alexmiguel95/"
          rel="noopener noreferrer"
          target="_blank"
        >
          Alex Miguel
        </a>
      </span>
    </header>
  </div>
</footer>
`;
